### ISSUE TITLE
randomASCII(length) function

### ISSUE THREAD

**Issue by filimonov** [ClickHouse/ClickHouse#5318]
Can be quite helpful for tests / creating fake data in some cases.

Workarounds are possible but quite complicated.

**Comment by alexey-milovidov** [comment]
I also have in my stash:

- randomString(length) - random binary string of specified length,
- fuzzBits(s, prob) - alternate bits in a string with desired probability, 
- fuzzBytes(s, prob) - alternate bytes in a string with desired probability.

**Comment by 4ertus2** [comment]
It’s also interesting to look at R sample() function that could generate samples with or without  replacement.

**Comment by BayoNet** [comment]
As a base functionality I suggest to use the following idea:

```
std::string get_askii_string(int string_len)
{
    //Generates numbers in the [32, 127] range uniformly and adds them as symbols to the result string.

    std::string resulting_string("");

    std::default_random_engine generator(time(0));
    std::uniform_int_distribution<int> distribution(32, 127);

    int i = 0;
    char charachter;

    while( i < string_len ){

        charachter = distribution(generator);
        if( isprint(charachter) ){
            resulting_string += charachter;
            i++;
        }
    }

    return resulting_string;
}
```

If Ok, I'll integrate it to the ClickHouse code.

**Comment by BayoNet** [comment]
Merged in #8401, #8451
